packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -49,7 +49,7 @@ class ModuleScopePlugin {
         )
       );
       // Error if in a parent directory of src/
-      if (requestRelative[0] === '.') {
+      if (requestRelative.startsWith('../')) {
```

Comment:
This should probably use `path.sep` instead of `/` for cross platform reasons. 😄
You could also be explicit and just add a second case for `\`.